docs/versioned-plugins/filters/http-v0.1.0.asciidoc

```diff
@@ -0,0 +1,337 @@
+:plugin: http
+:type: filter
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v0.1.0
+:release_date: 2018-12-12
+:changelog_url: https://github.com/logstash-plugins/logstash-filter-http/blob/v0.1.0/CHANGELOG.md
+:include_path: ../include/6.x
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="{version}-plugins-{type}s-{plugin}"]
+
+=== HTTP filter plugin {version}
+
+include::{include_path}/plugin_header.asciidoc[]
+
+==== Description
+
+The HTTP filter provides integration with external web services/REST APIs.
+
+[id="{version}-plugins-{type}s-{plugin}-options"]
+==== HTTP Filter Configuration Options
+
+This plugin supports the following configuration options plus the <<{version}-plugins-{type}s-{plugin}-common-options>> described later.
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<{version}-plugins-{type}s-{plugin}-body>> | String, Array or Hash |No
+| <<{version}-plugins-{type}s-{plugin}-body_format>> |{logstash-ref}/configuration-file-structure.html#string[string]|No
+| <<{version}-plugins-{type}s-{plugin}-headers>> |{logstash-ref}/configuration-file-structure.html#hash[hash]|No
+| <<{version}-plugins-{type}s-{plugin}-query>> |{logstash-ref}/configuration-file-structure.html#hash[hash]|No
```

Comment:
`query` is referenced, but not defined.

Reply:
@jsvd Hmmm...  I guess technically, these comments belong in the filters-http repo.

Reply:
Agreed, I've added these to https://github.com/logstash-plugins/logstash-filter-http/pull/7